akka-docs/_sphinx/themes/akka/static/warnOldDocs.js

```diff
@@ -0,0 +1,70 @@
+jQuery(document).ready(function ($) {
+
+  var latestInSeries = {
```

Comment:
Actually, this dict should be retrieved from the server at a static URL so that we can update it in one central place.